### ISSUE TITLE
DeucalionClient: Exception while setting up connection with Deucalion named pipe

### ISSUE THREAD

**Issue by seivan** [marzent/IINACT#54]
Getting an error, thought not sure if it's related to my system or the new patch that dropped.

```

FFXIV_ACT_Plugin: Process ffxiv_dx11 (212) Verified.
FFXIV_ACT_Plugin: Found Signature [ServerTime] at offset [02122FD8]
FFXIV_ACT_Plugin: Found Signature [MobArray] at offset [02169268]
FFXIV_ACT_Plugin: Found Signature [ChatLog] at offset [019AB2D0]
FFXIV_ACT_Plugin: Found Signature [MapID] at offset [020EE5F8]
FFXIV_ACT_Plugin: Found Signature [Player] at offset [02172A38]
FFXIV_ACT_Plugin: Found Signature [PartyList] at offset [0218B030]
FFXIV_ACT_Plugin: Found Signature [ZoneID] at offset [0213D178]
FFXIV_ACT_Plugin: Found VTable for [ChatLog] at address [260419C8]
DEBUG-MACHINA: DeucalionInjector: Successfully injected Deucalion dll into process id 212.
DEBUG-MACHINA: DeucalionClient: Exception while setting up connection with Deucalion named pipe.  Data will not be logged.  System.TimeoutException: The operation has timed out.
   at System.IO.Pipes.NamedPipeClientStream.ConnectInternal(Int32 timeout, CancellationToken cancellationToken, Int32 startTime)
   at Machina.FFXIV.Deucalion.DeucalionClient.Connect(Int32 processId)
```

**Comment by marzent** [comment]
I assume this is happening with the standalone version?

In general it was build in such a way to be compatible with future versions and this looks a bit like a Deucalion failure... With Deucalion especially it is a very bad idea to mix and match ACT and IINACT running on the same game process, even not at the same time, without restarting the game (or in general when Deucalion hangs up it is necessary to terminate the game process)

**Comment by seivan** [comment]
@marzent I am sorry, has something changed? Do I need to install anything else? 
I am not doing this on purpose, it's just what I had since previously. Had the setup for a DPS meter changed for XivOnMac?

**Comment by marzent** [comment]
Someone had the same issue on wine/linux with "real" ACT so I'm just gonna chalk it up to "not our bug"...

Yeah you can remove all traces of Wireshark, ChmodBPF and the log items they created if you use the plugin version of IINACT, Dalamud was also publicly released yesterday for 6.4 (if you can't wait that long and want to test things before you can also opt in to the plugin tester role on the Dalamud/XL discord)
